octopoes/octopoes/models/ooi/dns/records.py

```diff
@@ -185,3 +185,43 @@ class DNSCAARecord(DNSRecord):
     # without interior spaces or (2) a quoted string.
     value: str
     _natural_key_attrs = ["hostname", "flags", "tag", "value"]
+
+
+class DNSGPOSRecord(DNSRecord):
+    # RFC 1712
+    object_type: Literal["DNSGPOSRecord"] = "DNSGPOSRecord"
+    dns_record_type: Literal["GPOS"] = "GPOS"
+
+    latitude: str | None = None
+    longitude: str | None = None
+    altitude: str | None = None
```

Comment:
Can these be optional according to the specs?

Reply:
Can be optional.

Reply:
![saturn](https://github.com/user-attachments/assets/debc047d-5be3-4430-8f7d-f6cc8ee55555)